MetadataExtractor/Formats/Mpeg/MpegAudioTypeChecker.cs

```diff
@@ -23,12 +23,20 @@ internal sealed class MpegAudioTypeChecker : ITypeChecker
                 11 - Layer I
 
             Additional bits contain more information, but are not required for file type identification.
+
+            MP3 with ID3V2-Tags https://id3.org/id3v2.4.0-structure
+            MP3-File with ID3V2-Tagging at start
          */
 
         public int ByteCount => 3;
 
         public Util.FileType CheckType(byte[] bytes)
         {
+
+            // MP3-File with "ID3" at start
+            if (bytes[0] == 0x49 && bytes[1] == 0x44 && bytes[2] == 0x33)
+                return Util.FileType.Mp3;
```

Comment:
I think other types of file can start with ID3 too, though this is an improvement over what we had before.